packages/react-core/src/helpers/Popper/Popper.tsx

```diff
@@ -133,22 +154,22 @@ export const Popper: React.FunctionComponent<PopperProps> = ({
     }
   };
   React.useEffect(() => {
-    addEventListener(onMouseEnter, triggerElement, 'mouseenter');
-    addEventListener(onMouseLeave, triggerElement, 'mouseleave');
-    addEventListener(onFocus, triggerElement, 'focus');
-    addEventListener(onBlur, triggerElement, 'blur');
-    addEventListener(onTriggerClick, triggerElement, 'click');
-    addEventListener(onTriggerEnter, triggerElement, 'keydown');
+    addEventListener(onMouseEnter, refElement || triggerElement, 'mouseenter');
+    addEventListener(onMouseLeave, refElement || triggerElement, 'mouseleave');
+    addEventListener(onFocus, refElement || triggerElement, 'focus');
+    addEventListener(onBlur, refElement || triggerElement, 'blur');
+    addEventListener(onTriggerClick, refElement || triggerElement, 'click');
+    addEventListener(onTriggerEnter, refElement || triggerElement, 'keydown');
```

Comment:
DRY would be like:
```js
const popperElement = refElement || triggerElement; // this line can be included above React.useEffect
addEventListener(onMouseEnter, element, 'mouseenter');
...
```

Reply:
Thanks, popperElement is the element that pops up, so i've named it instead refOrTrigger